core/state/dump.go

```diff
@@ -55,31 +57,41 @@ func (db *DB) RawDump() Dump {
 		if err := rlp.DecodeBytes(it.Value, &data); err != nil {
 			panic(err)
 		}
-
 		obj := newObject(nil, common.BytesToAddress(addr), data)
+		var wrapper staking.ValidatorWrapper
+		wrap := ""
+		if err := rlp.DecodeBytes(obj.Code(db.db), &wrapper); err != nil {
```

Comment:
what is this code?

Reply:
makes debugging easy, dumping the validator wrapper if it exists in the code field